### PR TITLE
Apply wpautop to front-end display of HTML fields.

### PR DESCRIPTION
**Pull request by iamntz**

### CODE REVIEW

**Comment by Shelob9** [comment]
Good catch. I opened #2619 for this and edited the title to reflect the contents of the change.

File: fields/html/field.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 }else{
 	$html_template = $field[ 'config' ][ 'default' ];
-	$content = do_shortcode( Caldera_Forms::do_magic_tags(  $html_template   ) );
+	$content = do_shortcode( Caldera_Forms::do_magic_tags( wpautop( $html_template ) ) );
```
Comment (Shelob9) [inline fields/html/field.php]: @iamntz Not a blocker, but it concerns me as your  commit messages become part of our git history, that your commit message doesn't describe the reason for the commit, which happens here.

A better commit message would be something like
Applies wpautop formatting to html field front-end display #2619

**Comment by Shelob9** [comment]
I am merging as failed job https://travis-ci.org/CalderaWP/Caldera-Forms/jobs/390118886 is a Travis issue. Other tests passed.

**Comment by lyatziv** [comment]
This fix breaks any javascript that may be in a `$html_template`

**Pull request merged**